### PR TITLE
Native Queue Length

### PR DESCRIPTION
**Pull request by tmasternak**

### CODE REVIEW

File: tutorials/monitoring-setup/index.md [hunk 1]
```diff
@@ -204,8 +204,9 @@ You can then enable monitoring and configure the location of the monitoring queu
 
 snippet: SetupMonitoring-ConfigureMetrics
 
-NOTE: If you are using the MSMQ transport then you also need to specify the machine where the monitoring queue is located. i.e. `Particular.Monitoring@MACHINENAME`. This should be the same machine where your monitoring instance is installed.
+NOTE: When using the MSMQ transport monitoring queue machine name i.e. `Particular.Monitoring@MACHINENAME`. This should be the name of the machine that monitoring instance is installed on.
```
Comment (mikeminutillo) [inline tutorials/monitoring-setup/index.md]: I'm not sure this sentence is any clearer

File: monitoring/metrics/msmq-queue-length.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+---
+title: Setup Queue Length Metrics Reproting for the MSMQ Transport
+reviewed: 2018-05-11
+component: MetricsServiceControl
+related:
+  - samples/logging/metrics  
+---
+
+The component `NServiceBus.Metrics.ServiceControl.Msmq` monitors endpoints' queue length and passes that data to `NServiceBus.Metrics.ServiceControl` which in turn sends it to an instance of `ServiceControl.Monitoring` service.
```
Comment (mikeminutillo) [inline monitoring/metrics/msmq-queue-length.md]: Is it worth creating a mermaid diagram here to show these relationships?

File: tutorials/monitoring-setup/index.md [hunk 3]
```diff
@@ -204,8 +204,9 @@ You can then enable monitoring and configure the location of the monitoring queu
...
 
+NOTE: When using the MSMQ transport it is also necessary to install queue length metrics [reporter](/monitoring/metrics/msmq-queue-length.md) package.
```
Comment (mikeminutillo) [inline tutorials/monitoring-setup/index.md]: I wonder if this should be a snippet that can be pulled into multiple documents.

Alternatively, should we have a troubleshooting document or something? "Problem: Queue Length is not appearing. Solution: Make sure that the MSMQ Specific plugin is installed"

File: tutorials/monitoring-setup/index.md (hunk 1 above)
Comment (tmasternak) [reply tutorials/monitoring-setup/index.md]: 👍 reverted

File: monitoring/metrics/msmq-queue-length.md (hunk 2 above)
Comment (tmasternak) [reply monitoring/metrics/msmq-queue-length.md]: added

File: tutorials/monitoring-setup/index.md (hunk 3 above)
Comment (tmasternak) [reply tutorials/monitoring-setup/index.md]: I've added the troubleshooting guide for ql and liked from the setup guide.

**Comment by tmasternak** [comment]
@mikeminutillo I've addressed the review comments.

**Comment by tmasternak** [comment]
@mikeminutillo correct.

**Pull request merged**